src/content/1.7/modules/concepts/forms/admin-forms.md

```diff
@@ -1,8 +1,6 @@
 ---
-title: Alter forms on modern pages
+title: Admin forms on Admin pages
 weight: 4
-aliases:
-  - /1.7/modules/concepts/hooks/alter_forms_on_modern_pages
```

Comment:
You rename the page, you must keep the redirection for SEO.

Reply:
ok